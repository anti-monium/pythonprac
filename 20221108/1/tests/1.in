a = DivStr('jhbuyBUBBSObbnbdnuh')
d = DivStr('bruhbruhbruhbruhbruhbruhbruhbruhbruhbruhbruh')
print(d % 17 + d % 13 + d % 11 + a % 7 + a % 5)
print(*a // 4, *d % 14 // 2)
