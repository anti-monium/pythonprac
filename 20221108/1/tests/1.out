uhbruhbruhhbruhbdnuhdnuh
jhbu yBUB BSOb bnbd u h
